test/apps/react-routing/react-router-dom-v5-hash/README.md

```diff
@@ -0,0 +1,21 @@
+# Okta React Router DOM v5 Test App
+
+## Install
+```bash
+$ git clone https://github.com/okta/okta-react.git
+$ yarn
+$ cd ./test/apps/react-routing/react-router-dom-v5
+```
+
+## Configure
+Add an `.env` file with the following fields
```

Comment:
Do we want to stick with the `testenv` pattern in this repo, which might be easier for future CI integration?